src/CcdImage.cc

```diff
@@ -41,6 +41,8 @@ static double sq(const double &x) { return x*x;}
 
 void CcdImage::LoadCatalog(const lsst::afw::table::SortedCatalogT<lsst::afw::table::SourceRecord> &Cat,const std::string &fluxField)
 {
+  // TODO: these should use slots, e.g. "slot_Centroid_x", or better:
+  // self.config.centroidName + "_x" in python.
```

Comment:
I'll be curious what @TallJimbo has to say. My understanding was that slots were preferred in cases where the default would suffice, and in that situation the slot name could be hard coded. If the algorithm needs more flexibility then the next step is to use a config parameter to specify the field name (using the slot as the default if reasonable). I don't think retargeting a slot is ever expected to be done by an algorithm.